mutt/dump-ical.py

```diff
@@ -35,26 +35,22 @@ def pretty_print_invitation(invitation):
     end = event['dtend'][0].value
     location = event['location'][0].value
     description = event['description'][0].value
-    sequence = event['sequence'][0].value
-    print "="*70
-    if int(sequence) > 0:
-        print "MEETING UPDATE".center(70)
```

Comment:
Woops, seems like i lost this one. Sorry, will fix this later :-)

Am 24. August 2016 20:56:24 MESZ, schrieb Si Beaumont <notifications@github.com>:
>> @@ -35,26 +35,22 @@ def pretty_print_invitation(invitation):
>>      end = event['dtend'][0].value
>>      location = event['location'][0].value
>>      description = event['description'][0].value
>> -    sequence = event['sequence'][0].value
>> -    print "="*70
>> -    if int(sequence) > 0:
>> -        print "MEETING UPDATE".center(70)
>
>This was useful at least for me and has dropped off. If it's an update
>to an existing meeting then it's likely my calendar has already taken
>it into account (in the case of Exchange) and the email is just
>informational. This string lets me know it's OK to just delete the mail
>without any other action. However, if it's a new meeting then I need to
>go and accept it... worse, if I delete it then it doesn't appear in my
>calendar.